scripts/test_genesis_docker_compose.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -ex
 cd test_docker
+sleep 10
```

Comment:
I think if we need sleep it should be at a previous step. Or is this just a leftover for local testing?

Reply:
Yeah, sorry 😓